addons/html_builder/static/src/core/builder_options_plugin.js

```diff
@@ -65,12 +65,6 @@ export class BuilderOptionsPlugin extends Plugin {
             this.getResource("builder_header_middle_buttons")
```

Comment:
Commit message:
- the char limit is 72 chars, could you cut the lines to reach that number ? :slightly_smiling_face: